ci/cron/daily-compat.yml

```diff
@@ -281,6 +281,7 @@ jobs:
           gcs "$GCRED" cp "$OUT.tgz" "gs://daml-data/perf/http-json-oracle/${REPORT_ID}.tgz"
 
         displayName: http-json-oracle perf
+        name: out
```

Comment:
The variable is referenced as `out.oracle_perf_results` so the job needs to be named `out`.

Reply:
Is there some way in which we can catch these kind of mistakes earlier?

Reply:
Not an automated one I can think of, no. Running the job when it's a PR and checking the output is the only way I know of. Which I thought we'd done for this one, but I guess it slipped through.

Reply:
Looks like the [compat test on the branch failed](https://dev.azure.com/digitalasset/daml/_build/results?buildId=87510&view=results), so the actual print step never ran.

Best lesson I can think of here is if you've added a job _and changed another_, you should ensure _both_ succeed before merging.

Reply:
@garyverhaegen-da I had to merge this PR since I had broken compat-tests with the `declare -x fn` change so this was pushed before we completed the daily-compat tests, which takes hours to run. I was monitoring the channel but it seems this was an unfortunate case of other things breaking adding noise in our ways to test and react quickly.

Reply:
Right, so I guess another lesson we could take from this is to push bugfixes in their own separate PRs, rather than mixing them with new features.